ansible/deploy_common.yml

```diff
@@ -32,3 +32,16 @@
   hosts: all
   roles:
     - {role: monit, tags: monit}
+
+- name: Configure static routes
+  hosts: all
+  tasks:
+  - name: Adds static routes
+    shell: 'if [ `ip route show {{ item.prefix }} | wc -l` -eq "0" ];  then route add -net {{ item.prefix }} gw {{ item.gateway }}; else echo "already_exists"; fi'
```

Comment:
rather than doing this in one command can we do it in 2 where the first one checks if it's necessary and the 2nd one does the change?

Reply:
something like https://stackoverflow.com/a/35128533 ?